### ISSUE TITLE
Reports interfaces and abstract classes, should report only concrete classes only (IMO)

### ISSUE THREAD

**Issue by simPod** [thephpleague/tactician-phpstan#4]
Let's consider every command implements `Command` interface

```php
final class JustDoIt implements Command 
{
...
}
```

PhpStan Extension reports 

```
Tactician tried to route the command My\Namespace\Command to My\Namespace\CommandHandler::handle but while the class could be loaded, the method 'handle' could not be found on the class.
```

Also same for abstract commands (I have one, yes :( )

```php
abstract class DoIt implements Command 
{
...
}
```

**Comment by rosstuck** [comment]
Hmm, is this inside a function where the Command is only typehinted as a Command but not JustDoIt? 

Regardless, will try to fix this. Can you create a test or give me more info so I can create one?

**Comment by simPod** [comment]
@rosstuck added a test https://github.com/thephpleague/tactician-phpstan/pull/7

Found out it's because I typehint as `Command` or `DoIt`. 

My usecase is that I eg. materialize command in a queue consumer as `Command` because it supports all commands.

```php
public function consume(Message $message) : void
{
    /** @var Command $command */
    $command = $this->serializer->deserialize($message->payload, $message->getHeader(...), 'json');

    try {
        $this->commandBus->handle($command);
    } catch (Throwable $throwable) {
        $this->logger->error($throwable->getMessage(), ['exception' => $throwable]);
    }
}
```

**Comment by rosstuck** [comment]
Think I just fixed this in master now, can you confirm? :)

**Comment by simPod** [comment]
Lookin good, we can close :)
